bindings/netframework/ElectionGuard.NetFramework/ElectionGuard.NetFramework.x64.sln

```diff
@@ -0,0 +1,51 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.31129.286
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ElectionGuard.Tests", "Tests\ElectionGuard.Tests\ElectionGuard.Tests.csproj", "{3309CD0B-68E0-4769-A9AE-D795985BA53D}"
+EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "electionguard", "..\..\..\build\libs\msvc\x64\src\electionguard.vcxproj", "{4944669C-1FEB-3025-BDF6-901832F0D21E}"
```

Comment:
since cmake breaks out each of the builds into processor architectures, i had to do the same with the solution files